spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -1314,6 +1321,82 @@ public void setOptions(Map<String, String> options) {
 
 	}
 
+	public static class RetryTopic {
+
+		private Integer attempts;
```

Comment:
This should have field Javadoc like other properties in this class. Please review the to phrase it consistently.